apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/CaseFiles/CaseFiles.tsx

```diff
@@ -19,6 +19,7 @@ export interface PoliceCaseFilesData {
   files: PoliceCaseFile[]
   isLoading: boolean
   hasError: boolean
+  errorMessage?: string
```

Comment:
This errorMessage is unused, isn't it missing in setPoliceCaseFiles ?

Reply:
I believe we're using the errorMessage in line 209 in caseFilesForm

```
policeCaseFiles?.hasError ? (
    policeCaseFiles?.errorMessage &&
    policeCaseFiles?.errorMessage.indexOf('404') > -1 ? (
...
```

Am I missing something?

Reply:
This was resolved irl